### PR TITLE
Added Hudson Rock to Credentials section

### PR DESCRIPTION
**Pull request by hudsonrock-partnerships**
Free cybercrime tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -318,6 +318,7 @@ A curated list of awesome search engines useful during Penetration testing, Vuln
 - [LeakCheck.io](https://leakcheck.io/) - Make sure your credentials haven't been compromised
 - [crackstation.net](https://crackstation.net/) -Massive pre-computed lookup tables to crack password hashes
 - [HashKiller](https://hashkiller.io/listmanager) - Pre-cracked Hashes, easily searchable
+- [Hudson Rock](https://www.hudsonrock.com/threat-intelligence-cybercrime-tools) - Check if a username, email, or domain was compromised in global Infostealer attacks
```
Comment (edoardottt) [inline README.md:321]: Maybe this description would be better: `Use Hudson Rock’s free cybercrime intelligence tools to learn how compromised credentials are impacting your business`

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (edoardottt) [reply README.md:321]: ok, waiting for the changes

**Review by hudsonrock-partnerships — COMMENTED** [review]
Changed the description

**Review by edoardottt — APPROVED** [review]
lgtm

**Comment by edoardottt** [comment]
thanks @hudsonrock-partnerships !

**Pull request merged**